components/Button/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react'
+import {StyleSheet, View} from 'react-native'
+import PropTypes from 'prop-types'
+
+export default class Button extends Component {
+    static propTypes = {
+        text: PropTypes.string,
```

Comment:
Now that you actually use `this.props.text`, you should probably mark this as required. Even though you give a default prop, this helps for self-documentation